src/appengine/libs/handler_flask.py

```diff
@@ -21,12 +21,14 @@
 import re
 import requests
 
+from flask import g
```

Comment:
remove ?

Reply:
This is needed for per-request caching: https://flask.palletsprojects.com/en/1.1.x/api/#flask.g

Reply:
:)